aliases.yaml

```
@@ -77,8 +77,18 @@ cmds:
     cmd: describe
   - short: g
     cmd: get
+  - short: g
+    cmd: get
+    suffix:
+      short: g
+      cmd: " -o wide | grep"
   - short: ga
     cmd: "get --all-namespaces -o wide"
+  - short: ga
+    cmd: "get --all-namespaces -o wide"
+    suffix:
+      short: g
+      cmd: " -o wide | grep"
```

We duplicate `-o wide`
```suggestion
      cmd: " | grep"
```